types/coin.go

```diff
@@ -11,40 +11,33 @@ import (
 )
 
 const (
-	// AttoEvmos defines the default coin denomination used in Evmos in:
+	// BaseDenom defines the default coin denomination used in Evmos in:
 	//
 	// - Staking parameters: denomination used as stake in the dPoS chain
 	// - Mint parameters: denomination minted due to fee distribution rewards
 	// - Governance parameters: denomination used for spam prevention in proposal deposits
 	// - Crisis parameters: constant fee denomination used for spam prevention to check broken invariant
 	// - EVM parameters: denomination used for running EVM state transitions in Evmos.
-	AttoEvmos string = "aevmos"
+	BaseDenom        string = "aevmos"
+	BaseDenomTestnet string = "atevmos"
 
 	// BaseDenomUnit defines the base denomination unit for Evmos.
 	// 1 evmos = 1x10^{BaseDenomUnit} aevmos
 	BaseDenomUnit = 18
 
+	// DisplayDenom defines the denomination displayed to users in client applications.
+	DisplayDenom        string = "evmos"
+	DisplayDenomTestnet string = "tevmos"
```

Comment:
Can we have this in a more structured way? like a map of configs by chain-id? something silimar to [geth](https://github.com/ethereum/go-ethereum/blob/deb5c087c41e5d7692fac2cff4c5aa547a108bc1/params/config.go#L40)
So we are not restricted with the utils.IsTesnet() when setting the config. Will be just checking if the chain id is within the available configs and set the corresponding config.

Reply:
agree, it would be better